index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Youtube clone</title>
+
+  <link rel="icon" href="images/favicon.ico" type="image/x-icon">
+
+  <link
+    href="css/tags.css"
+    rel="stylesheet"
+  />
+  <link
+    href="css/classes.css"
+    rel="stylesheet"
+  />  
+  <link
+  href="css/elements.css"
+  rel="stylesheet"
+  />
+  <link
+  href="css/highlight.css"
+  rel="stylesheet"
+  />  
```

Comment:
There is no file highlight.css, so you need to remove this import.

Reply:
Thanks for the constructive feedback! Yes, we noticed that there was the file link left in the index.html. We used it for highlighting the page structure. Its removed now.
Thanks!